modules/base/mw.cx.SiteMapper.js

```
@@ -189,7 +208,8 @@ mw.cx.SiteMapper.prototype.getLanguagePairs = function () {
 			.then((response) => response.json())
 			.then((response) => ({
 				targetLanguages: response.target,
-				sourceLanguages: response.source
+				sourceLanguages: ["en", "ar"],
```

ESLint

[error] A space is required after '['.

(array-bracket-spacing)